flow-typed/npm/react-navigation_v1.x.x.js

```diff
@@ -717,7 +717,7 @@ declare module 'react-navigation' {
       toString: () => string,
     },
     init: {
-      (payload: { params?: NavigationParams }): NavigationInitAction,
+      (payload?: { params?: NavigationParams }): NavigationInitAction,
```

Comment:
I checked the `init` function and it doesn't require me to pass any `payload`

Reply:
This libdef is meant to be downloaded from the `flow-typed` repo. You’ll need to submit two additional PRs updating this: one to the main `react-navigation` repo, and one to the `flow-typed` repo.